packages/data-context/src/actions/ElectronActions.ts

```diff
@@ -104,4 +106,14 @@ export class ElectronActions {
       return obj.filePath || null
     })
   }
+
+  showSystemNotification (title: string, body: string) {
+    const notification = this.ctx.electronApi.createNotification(title, body, !this.isMac ? getPathToIcon('icon_64x64.png') : undefined)
```

Comment:
Could probably pull the `getPathToIcon` out as a top-level constant since it won't change between invocations

Reply:
Yeah I think I'm going to take this out for now